packages/venia-ui/package.json

```diff
@@ -44,7 +44,7 @@
     "memoize-one": "~5.0.0",
     "prop-types": "~15.7.2",
     "react-feather": "~2.0.3",
-    "react-head": "~3.1.1",
+    "react-helmet-async": "^1.0.2",
```

Comment:
@jimbo - I know we prefer tilde over caret, but recall this was copied from a Storybook dep; do we still want tilde here to be consistent?